### ISSUE TITLE
^C broken on launch?

### ISSUE THREAD

**Issue by deliciouslytyped** [gcla/termshark#120]
I'm running 2.2.0. I haven't really been able to figure out what the underlying issue is, 
Here is an example:
```
# termshark -i eth0
(The termshark UI will start when packets are detected...)
^C^C^C <does nothing>
```
I think it may be timing related but I'm not sure; if I press ^C shortly after startup and reaching the waiting for packets dialog, any amount of ^C does nothing. Other times (if you wait long enough before the first ^C?) it works.

**Comment by gcla** [comment]
Yikes. I see this too. Thanks - I'll get a fix out for it.

**Comment by gcla** [comment]
Hi @deliciouslytyped - if you have time, I've pushed commit 2461481468905cde4187eba7a245df9e026b3848, an attempt to fix this intermittent ctrl-c problem.

**Comment by gcla** [comment]
I'll close this for now. I hope it's fixed as of bfea053d7821d2c5c670b599df48f4c346fe4079. Please do re-open if it recurs.
